src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
```

Comment:
Ensure that the 'table' element exists before accessing its rows. Consider adding a check to see if 'table' is not null to avoid potential runtime errors.